lib/include/event-driven/vtsHelper.h

```diff
@@ -68,11 +68,18 @@ class vtsHelper {
         return dt;
     }
 
+    static double deltaS(const int current_tick, const int prev_tick)
+    {
+        return deltaTicks(current_tick, prev_tick) * tsscaler;
+    }
+
     static double deltaMS(const int current_tick, const int prev_tick)
     {
-        return deltaTicks(current_tick, prev_tick) * tsscaler * 1000.0;
+        return deltaS(current_tick, prev_tick) * 1000.0;
     }
 
+
```

Comment:
Knowing your attention to code aesthetics, you might want to leave fewer empty lines here?

Reply:
done